samples/duckdb/faa_duckdb/flights_2.malloy

```diff
@@ -1,14 +1,32 @@
-import "flights.malloy"
 
 // carrier analysis.
 //  show where carrires fly, what kinds of planes they use
+// 
+// over view of carriers
+//   What percentage the market does a carrier have?
```

Comment:
nit: Missing "of" 
What percentage of the market does a carrier have?

Reply:
thanks